DungeonAPI/map.py

```diff
@@ -0,0 +1,84 @@
+import random
+import time
+from DungeonAPI.room import Room
+
+class Map:
+    def __init__(self, size, room_limit):
+        self.grid = []
+        row = [0] * size
+        for i in range(size):
+            row = row.copy()
+            self.grid.append(row)
+        self.center     = size//2
+        self.room_count = 0
+        self.size       = size
+        self.room_limit = room_limit
+        self.rooms      = dict()
+        self.set_room(self.center, self.center)
+
+    def create_room(self, y, x):
+        id          = int(f"{str(y)}{str(x)}")
+        name        = f"Room #{id}"
+        description = f"The description for {name}."
+        return Room(None, name, description, (y, x), id)
+
+    def set_room(self, y, x):
+        if self.grid[y][x] != 1:
+            self.grid[y][x] = 1
+            self.room_count += 1
+            self.rooms.update({(y, x): self.create_room(y, x)})
```

Comment:
Can we changes these two tuples to `(x, y)` instead of `(y, x)` ? 

This will make the room locations match with the tuples in the rest of the back-end so far '' '

Reply:
Changed in commit hash 4324938 👍